### ISSUE TITLE
Accessibility 9.1.3.1e - data tables correctly structured

### ISSUE THREAD

**Issue by hassmal** [Dataport/terminfinder-frontend#310]
# Bug

Es fehlt eine (sichtbare) Tabellenüberschrift in der Tabelle zur Übersicht der eingetragenen Termine. Diese ist programmatisch zudem nicht korrekt aufgebaut, es fehlt ein Tabellenheader. Insgesamt wirkt die Darstellung der Termine nicht übersichtlich und wird vom Screenreader nicht optimal vorgelesen.
Beispiel für Screenreaderausgabe für Screenshot „Feb Achtundzwanzig M I 12 Uhr März Null Acht F R 14 Uhr“

## SOLL-Zustand

- Eine sichtbare Headline `<h3>` „Termin-Übersicht“ ergänzen. Falls dies nicht gewünscht ist, sollte eine (ausgeblendete) Tabellenüberschrift mittels `<caption>` für den Screenreader ergänzt werden.
- Einen Tabellenkopf `<th>` ergänzen, z.B. als Spalte davor. Mittels „scope“ sollte der Tabellenkopf mit den Inhalten verknüpfet werden.
- Die Bereiche Datum und Uhrzeit könnten als extra Zeile dabei angelegt werden. Falls keine Zeitangabe zuvor ausgewählt wurde, könnte als Default-Wert „Ganztägig“ in dem Feld in der Reihe „Uhrzeit“ dann stehen.

Layoutbeispiel:
![grafik](https://github.com/user-attachments/assets/f762a54c-46b7-4ede-b8b3-314cfa8ca521)

------------

# Bug

A (visible) table header is missing in the table for the overview of entered appointments. This is also not correctly structured programmatically, a table header is missing. Overall, the display of the appointments is not clear and is not read out optimally by the screen reader.
Example of screen reader output for screenshot “Feb Twenty eight M I 12 o'clock March Zero eight F R 14 o'clock”

## Desired state

- Add a visible headline `<h3>` “Appointment overview”. If this is not desired, a (hidden) table heading should be added using `<caption>` for the screen reader.
- Add a table header `<th>`, e.g. as a column in front. The table header should be linked to the content using “scope”.
- The date and time areas could be created as an extra line. If no time has been selected beforehand, the default value “All day” could then be entered in the field in the “Time” row.

Layout example:
![grafik](https://github.com/user-attachments/assets/f762a54c-46b7-4ede-b8b3-314cfa8ca521)
